### ISSUE TITLE
The last character of some links are not linkified

### ISSUE THREAD

**Issue by imrehg** [petrosagg/papagal#3]
## Expected behaviour

All the links in the text work as intended,

## Actual behaviour

Some links have their last character not become part of the URL, and thus not linkified, resulting in invalid links, or linking to the wrong place in general

## Info

* Papagal version 1.6.5
* Browser: Chromium 75.0.3770.100 (Official Build) Arch Linux (64-bit)

## Screenshots

The screenshot with link and hover in an [example thread](https://www.flowdock.com/app/rulemotion/resin-starters/threads/a1nHBUzbwlPYBnoCKB9xY19_KDG) when using Papagal:

![Screenshot_2019-07-03_10-12-19](https://user-images.githubusercontent.com/38863/60580025-3c055a00-9d7c-11e9-88ea-0b35f47c98b7.png)

When Papagal's turned off, the same link is correct:

![Screenshot_2019-07-03_10-16-27](https://user-images.githubusercontent.com/38863/60580036-41fb3b00-9d7c-11e9-9053-4f7d05ac20a4.png)

**Comment by petrosagg** [comment]
Interestingly, this happens only on chrome/chromium. I updated the code from upstream and created a new version that seems to work. I didn't dig deep into why it happened though, it looks related to this change from upstream:

https://github.com/petrosagg/papagal/commit/f4f4e678c81f014a52f2055caa7cba4844d67d11#diff-44f732579ad37a7f6d788a7a8cd92255  (see file src/vendor/markdown-it-flowdock/url.js)

I pushed version 1.6.6 in the repo that seems to fix the issue. I'll let this running on my machine for a day or two and if there are no issues I'll publish to the extension store.

@imrehg I'd appreciate if you confirm this fixes the issue. To test yourself just do a `npm run build` on the repo and load the unpacked extension from `dist/chromium/`
